src/schemas/map.json

```diff
@@ -161,7 +191,7 @@
       "type": "integer",
       "minimum": 0
     },
-    "ToleratedFailurePath": {
+    "ToleratedFailureCountPath": {
```

Comment:
this as a copy/paste error

it was correct in the checkers but incorrect here.